src/components/Error/Error.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useEffect } from 'react';
+import './Error.scss';
+
+type Props = {
+  getDataError: boolean,
+  postDataError: boolean,
+  deleteDataError: boolean,
+  inputState: boolean,
+  disableErrorHandling: () => void,
+};
+
+export const Error: React.FC<Props> = ({
+  getDataError,
+  postDataError,
+  deleteDataError,
+  inputState,
+  disableErrorHandling,
+}) => {
+  const [timerId, setTimerId] = useState<NodeJS.Timeout | null>(null);
+
+  useEffect(() => {
+    const id = setTimeout(() => disableErrorHandling(), 3000);
+
+    setTimerId(id);
+
+    return () => {
+      if (timerId) {
+        clearTimeout(timerId);
+      }
+    };
+  }, []);
+
+  return (
+    <div className="notification is-danger is-light has-text-weight-normal">
+      <button
+        type="button"
+        className="delete"
+        aria-label="delete"
+        onClick={() => {
```

Comment:
Move the onClick logic to a separate function. It follows the best practice of separating concerns and keeping your JSX code clean and focused on rendering components rather than defining complex logic directly within it.

Reply:
Fixed